### PR TITLE
add 78key version of 5x7 layout.

### PR DESCRIPTION
**Pull request by dedalo**
## Description

Added a 78 key layout to the dactyl manuform handwired keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/78key/78key.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avR/io.h>
+  #include <avR/inteRRupt.h>
+#endif
+#endif
+
+#define LAYOUT_78key( \
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/78key/78key.h:14]: I hate this style of layout.   Ideally, it should be more representative of the actual layout.  

Eg, like: 
https://github.com/qmk/qmk_firmware/blob/2cebac6f2f7f5a49adf7f2ce8a94c8ba1dbef707/keyboards/handwired/dactyl_manuform/5x6/5x6.h#L14-L23

File: .gitignore [hunk 2]
```diff
@@ -49,12 +49,7 @@ doxygen/
 .browse.VC.db*
 *.stackdump
 # Let these ones be user specific, since we have so many different configurations
-.vscode/c_cpp_properties.json
-.vscode/launch.json
-.vscode/tasks.json
-.vscode/last.sql
-.vscode/temp.sql
-.vscode/ipch/
+.vscode
```
[the remark this answers is not in the document]
Comment (dedalo) [reply .gitignore]: done!

File: .gitignore [hunk 3]
```diff
@@ -73,3 +68,4 @@ __pycache__
 
 # prerequisites for updating ChibiOS
 /util/fmpp*
+keyboards/handwired/dactyl_manuform/5x7/.vscode/c_cpp_properties.json
```
[the remark this answers is not in the document]
Comment (dedalo) [reply .gitignore]: done!

File: keyboards/handwired/dactyl_manuform/78key/78key.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    // // green led on
+    // DDRD |= (1<<5);
+    // PORTD &= ~(1<<5);
+
+    // // orange led on
+    // DDRB |= (1<<0);
+    // PORTB &= ~(1<<0);
+
+	matrix_init_user();
+};
```
[the remark this answers is not in the document]
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/78key.c]: done !

File: keyboards/handwired/dactyl_manuform/78key/78key.h [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+#include "dactyl_manuform.h"
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avR/io.h>
+  #include <avR/inteRRupt.h>
+#endif
+#endif
```
[the remark this answers is not in the document]
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/78key.h]: done!

File: keyboards/handwired/dactyl_manuform/78key/config.h [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+
+#define PRODUCT         Dactyl-Manuform (Ergodox)
```
[the remark this answers is not in the document]
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/config.h]: done!

File: keyboards/handwired/dactyl_manuform/78key/info.json [hunk 7]
```diff
@@ -0,0 +1,92 @@
+{
+    "keyboard_name": "Dactyl Manuform 5x7 (78 keys)",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 17,
+    "height": 9,
+    "layouts": {
+        "LAYOUT_5x7_78KEY": {
```
[the remark this answers is not in the document]
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/info.json]: done!

File: keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _FN     1
+#define _NUMPAD 2
```
[the remark this answers is not in the document]
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c]: done!

File: keyboards/handwired/dactyl_manuform/78key/78key.h (hunk 1 above)
Comment (dedalo) [reply keyboards/handwired/dactyl_manuform/78key/78key.h:2]: copied mine from the current 5x7, I like this too, but no sure how to redo all of it, I guess I need to change keymap.c too?

File: keyboards/handwired/dactyl_manuform/78key/78key.h [hunk 9]
```diff
@@ -1,33 +1,21 @@
...
-#include "dactyl_manuform.h"
-#include "quantum.h"
-
-#ifdef USE_I2C
-#include <stddef.h>
-#ifdef __AVR__
-  #include <avR/io.h>
-  #include <avR/inteRRupt.h>
-#endif
-#endif
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/78key/78key.h:1]:
```suggestion
#pragma once

#include "dactyl_manuform.h"
#include "quantum.h"
```

File: keyboards/handwired/dactyl_manuform/78key/78key.h (hunk 1 above)
Comment (noroadsleft) [reply keyboards/handwired/dactyl_manuform/78key/78key.h:5]: Yes, your `keymap.c` file would also need to be edited.

File: keyboards/handwired/dactyl_manuform/78key/78key.h [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    { L31,   L32,   L33, L34, L35, L36,   L37   }, \
+    { L41,   L42,   L43, L44, L45, L46,   L47   }, \
+    { L51,   L52,   L53, L54, L55, L56,   L57   }, \
+    { KC_NO, KC_NO, KC_NO, L64, L65, L66, L67   }, \
+    { R11,   R12,   R13, R14, R15, R16,   R17   }, \
+    { R21,   R22,   R23, R24, R25, R26,   R27   }, \
+    { R31,   R32,   R33, R34, R35, R36,   R37   }, \
+    { R41,   R42,   R43, R44, R45, R46,   R47   }, \
+    { R51,   R52,   R53, R54, R55, R56,   R57   }, \
+    { R61,   R62,   R63, R64, KC_NO, KC_NO, KC_NO }\
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/78key.h:37]: Changes this to the side-by-side formatting, per drashna:

```suggestion
    L11, L12, L13, L14, L15, L16, L17,    R11, R12, R13, R14, R15, R16, R17, \
    L21, L22, L23, L24, L25, L26, L27,    R21, R22, R23, R24, R25, R26, R27, \
    L31, L32, L33, L34, L35, L36, L37,    R31, R32, R33, R34, R35, R36, R37, \
    L41, L42, L43, L44, L45, L46, L47,    R41, R42, R43, R44, R45, R46, R47, \
    L51, L52, L53, L54, L55,                        R53, R54, R55, R56, R57, \
                               L56,         R52,                             \
                               L57,         R51,                             \
                             L64, L67,    R61, R64,                          \
                             L65, L66,    R62, R63                           \
  ) \
  { \
    { L11,   L12,   L13,   L14, L15,   L16,   L17   }, \
    { L21,   L22,   L23,   L24, L25,   L26,   L27   }, \
    { L31,   L32,   L33,   L34, L35,   L36,   L37   }, \
    { L41,   L42,   L43,   L44, L45,   L46,   L47   }, \
    { L51,   L52,   L53,   L54, L55,   L56,   L57   }, \
    { KC_NO, KC_NO, KC_NO, L64, L65,   L66,   L67   }, \
    { R11,   R12,   R13,   R14, R15,   R16,   R17   }, \
    { R21,   R22,   R23,   R24, R25,   R26,   R27   }, \
    { R31,   R32,   R33,   R34, R35,   R36,   R37   }, \
    { R41,   R42,   R43,   R44, R45,   R46,   R47   }, \
    { R51,   R52,   R53,   R54, R55,   R56,   R57   }, \
    { R61,   R62,   R63,   R64, KC_NO, KC_NO, KC_NO }\
```

File: keyboards/handwired/dactyl_manuform/78key/info.json [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+                {"label":"R54", "x":13, "y":4},
+                {"label":"R55", "x":14, "y":4},
+                {"label":"R56", "x":15, "y":4},
+                {"label":"R57", "x":16, "y":4},
+                {"label":"R51", "x":9,  "y":5, "w":2},
+                {"label":"R52", "x":9,  "y":6, "w":2},
+                {"label":"R61", "x":9,  "y":7},
+                {"label":"R62", "x":9,  "y":8},
+                {"label":"R63", "x":10, "y":8},
+                {"label":"R64", "x":10, "y":7}
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/info.json:88]:
```suggestion
                {"label":"L11", "x":0, "y":0},
                {"label":"L12", "x":1, "y":0},
                {"label":"L13", "x":2, "y":0},
                {"label":"L14", "x":3, "y":0},
                {"label":"L15", "x":4, "y":0},
                {"label":"L16", "x":5, "y":0},
                {"label":"L17", "x":6, "y":0},

                {"label":"R11", "x":10, "y":0},
                {"label":"R12", "x":11, "y":0},
                {"label":"R13", "x":12, "y":0},
                {"label":"R14", "x":13, "y":0},
                {"label":"R15", "x":14, "y":0},
                {"label":"R16", "x":15, "y":0},
                {"label":"R17", "x":16, "y":0},

                {"label":"L21", "x":0, "y":1},
                {"label":"L22", "x":1, "y":1},
                {"label":"L23", "x":2, "y":1},
                {"label":"L24", "x":3, "y":1},
                {"label":"L25", "x":4, "y":1},
                {"label":"L26", "x":5, "y":1},
                {"label":"L27", "x":6, "y":1},

                {"label":"R21", "x":10, "y":1},
                {"label":"R22", "x":11, "y":1},
                {"label":"R23", "x":12, "y":1},
                {"label":"R24", "x":13, "y":1},
                {"label":"R25", "x":14, "y":1},
                {"label":"R26", "x":15, "y":1},
                {"label":"R27", "x":16, "y":1},

                {"label":"L31", "x":0, "y":2},
                {"label":"L32", "x":1, "y":2},
                {"label":"L33", "x":2, "y":2},
                {"label":"L34", "x":3, "y":2},
                {"label":"L35", "x":4, "y":2},
                {"label":"L36", "x":5, "y":2},
                {"label":"L37", "x":6, "y":2},

                {"label":"R31", "x":10, "y":2},
                {"label":"R32", "x":11, "y":2},
                {"label":"R33", "x":12, "y":2},
                {"label":"R34", "x":13, "y":2},
                {"label":"R35", "x":14, "y":2},
                {"label":"R36", "x":15, "y":2},
                {"label":"R37", "x":16, "y":2},

                {"label":"L41", "x":0, "y":3},
                {"label":"L42", "x":1, "y":3},
                {"label":"L43", "x":2, "y":3},
                {"label":"L44", "x":3, "y":3},
                {"label":"L45", "x":4, "y":3},
                {"label":"L46", "x":5, "y":3},
                {"label":"L47", "x":6, "y":3},

                {"label":"R41", "x":11, "y":3},
                {"label":"R42", "x":11, "y":3},
                {"label":"R43", "x":12, "y":3},
                {"label":"R44", "x":13, "y":3},
                {"label":"R45", "x":14, "y":3},
                {"label":"R46", "x":15, "y":3},
                {"label":"R47", "x":16, "y":3},

                {"label":"L51", "x":0, "y":4},
                {"label":"L52", "x":1, "y":4},
                {"label":"L53", "x":2, "y":4},
                {"label":"L54", "x":3, "y":4},
                {"label":"L55", "x":4, "y":4},

                {"label":"R53", "x":13, "y":4},
                {"label":"R54", "x":13, "y":4},
                {"label":"R55", "x":14, "y":4},
                {"label":"R56", "x":15, "y":4},
                {"label":"R57", "x":16, "y":4},

                {"label":"L56", "x":6, "y":5, "w":2},
                {"label":"R52", "x":9,  "y":5, "w":2},

                {"label":"L57", "x":6, "y":6, "w":2},
                {"label":"R51", "x":9,  "y":6, "w":2},

                {"label":"L64", "x":6, "y":7},
                {"label":"L67", "x":7, "y":7},
                {"label":"R61", "x":9,  "y":7},
                {"label":"R64", "x":10, "y":7},

                {"label":"L65", "x":6, "y":8},
                {"label":"L66", "x":7, "y":8},
                {"label":"R62", "x":9,  "y":8},
                {"label":"R63", "x":10, "y":8}
```

File: keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+        // right hand
+                     KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,   KC_GRV,
+                     KC_Y,    KC_U,    KC_I,    KC_O,     KC_P,     KC_SLSH,  KC_PAUS,
+                     KC_H,    KC_J,    KC_K,    KC_L,     KC_SCLN,  KC_QUOT,  KC_PSCR,
+                     KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_LBRC,  KC_RBRC,  KC_INS,
+                                       KC_LEFT, KC_UP,    KC_DOWN,  KC_RGHT,  KC_BSLS,
+            KC_SPC,
+            KC_ENT,
+        OSM(MOD_RSFT), KC_CAPS,
+        KC_RCTL, KC_RALT),
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c:76]: Reformatting the keymap to side-by-side.

```suggestion
    // left hand                                                                       // right hand
    KC_ESC,        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,                  KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,
    KC_TAB,        KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    TT(1),                 KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_SLSH, KC_PAUS,
    KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    TT(2),                 KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_PSCR,
    OSM(MOD_LSFT), KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    LCTL(LALT(KC_DEL)),    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_LBRC, KC_RBRC, KC_INS,
    KC_LCTL,       KC_LALT, KC_LGUI, KC_RALT, KC_APP,                                                    KC_LEFT, KC_UP,   KC_DOWN, KC_RGHT, KC_BSLS,
                                                           KC_BSPC,                        KC_SPC,
                                                           KC_DEL,                         KC_ENT,
                                                       KC_END,  KC_HOME,               OSM(MOD_RSFT), KC_CAPS,
                                                       KC_PGDN, KC_PGUP,               KC_RCTL, KC_RALT
    ),
```

File: keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+        // right hand
+                     KC_F7,     KC_F8,     KC_F9,     KC_F10,    KC_F11,    KC_F12,    _______,
+                     _______,   XP(uACUTE,UACUTE),   XP(iACUTE,IACUTE),   XP(oACUTE,OACUTE),   _______,   _______,   _______,
+                     _______,   _______,   _______,   _______,   _______,   _______,   _______,
+                     XP(nTILDE,NTILDE),   _______,   _______,   _______,   _______,   _______,   _______,
+                                           _______,   _______,   _______,   _______,   _______,
+            _______,
+            _______,
+        _______, _______,
+        _______, _______),
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c:98]:
```suggestion
    // left hand                                                                           // right hand
    _______, KC_F1,             KC_F2,   KC_F3,             KC_F4,   KC_F5,   KC_F6,       KC_F7,             KC_F8,             KC_F9,             KC_F10,            KC_F11,  KC_F12,  _______,
    _______, _______,           _______, XP(eACUTE,EACUTE), _______, _______, _______,     _______,           XP(uACUTE,UACUTE), XP(iACUTE,IACUTE), XP(oACUTE,OACUTE), _______, _______, _______,
    _______, XP(aACUTE,AACUTE), _______, _______,           _______, _______, _______,     _______,           _______,           _______,           _______,           _______, _______, _______,
    _______, _______,           _______, _______,           _______, _______, RESET,       XP(nTILDE,NTILDE), _______,           _______,           _______,           _______, _______, _______,
    _______, _______,           _______, _______,           _______,                                                             _______,           _______,           _______, _______, _______,
                                                                         _______,              _______,
                                                                         _______,              _______,
                                                                     _______, _______,     _______, _______,
                                                                     _______, _______,     _______, _______
    ),
```

File: keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+        // right hand
+                     _______,   _______,   KC_NLCK,   _______,   KC_PMNS,   KC_PPLS,   _______,
+                     _______,   _______,   KC_P7,     KC_P8,     KC_P9,     _______,   _______,
+                     _______,   _______,   KC_P4,     KC_P5,     KC_P6,     KC_PAST,   _______,
+                     _______,   _______,   KC_P1,     KC_P2,     KC_P3,     KC_PSLS,   _______,
+                                           _______,   KC_P0,     KC_PDOT,   _______,   _______,
+            _______,
+            KC_PENT,
+        _______, _______,
+        _______, _______),
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/keymaps/default/keymap.c:120]:
```suggestion
    // left hand                                                      // right hand
    _______, _______, _______, _______, _______, _______, _______,    _______, _______, KC_NLCK, _______, KC_PMNS, KC_PPLS, _______,
    _______, _______, _______, _______, _______, _______, _______,    _______, _______, KC_P7,   KC_P8,   KC_P9,   _______, _______,
    _______, _______, _______, _______, _______, _______, _______,    _______, _______, KC_P4,   KC_P5,   KC_P6,   KC_PAST, _______,
    _______, _______, _______, _______, _______, _______, _______,    _______, _______, KC_P1,   KC_P2,   KC_P3,   KC_PSLS, _______,
    _______, _______, _______, _______, _______,                                        _______, KC_P0,   KC_PDOT, _______, _______,
                                                     _______,             _______,
                                                     _______,             KC_PENT,
                                                 _______, _______,    _______, _______,
                                                 _______, _______,    _______, _______
    ),
```

File: keyboards/handwired/dactyl_manuform/78key/rules.mk [hunk 15]
```diff
@@ -0,0 +1,20 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/rules.mk:4]:
```suggestion
#   change yes to no to disable
```

File: keyboards/handwired/dactyl_manuform/78key/rules.mk [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/rules.mk:9]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
```

File: keyboards/handwired/dactyl_manuform/78key/rules.mk [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/rules.mk:11]:
```suggestion
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover 
```

File: keyboards/handwired/dactyl_manuform/78key/rules.mk [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODEMAP_ENABLE = yes         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (noroadsleft) [inline keyboards/handwired/dactyl_manuform/78key/rules.mk:20]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
UNICODEMAP_ENABLE = yes     # Unicode
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/9646/files).

**Pull request closed without merging**